packages/core/svelte/package.json

```
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@solana/wallet-adapter-svelte",
-    "version": "0.1.0",
+    "version": "0.2.0",
```

```suggestion
    "version": "0.1.0",
```

Don't need to bump this version, the package is private and unpublished.